### PR TITLE
Weather stations dropdown with complete integration tests

### PR DESCRIPTION
**Pull request by Kyubinhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 Wildfire Predictive Services to support decision making in prevention, preparedness, response and recovery.
 
-
 ## Getting Started
 
 ### Dependencies
 
-* [Node.js](https://nodejs.org/en/) - You’ll need to have 10.16.0 or later version on your local development machine. You can use [nvm](https://github.com/nvm-sh/nvm#installation) (macOS/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows#node-version-manager-nvm-for-windows) to switch Node versions between different projects.
+- [Node.js](https://nodejs.org/en/) - You’ll need to have 10.16.0 or later version on your local development machine. You can use [nvm](https://github.com/nvm-sh/nvm#installation) (macOS/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows#node-version-manager-nvm-for-windows) to switch Node versions between different projects.
```
Comment (Sybrand) [inline README.md]: Do we know that 10.16.* is what we'll have running on our build server? @DerekRoberts ?

File: src/api/axios.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
+
+const isBundled = process.env.NODE_ENV === 'production'
+const prod = 'https://wps-api-dev-6-secure-auzhsi.pathfinder.gov.bc.ca'
```
Comment (Sybrand) [inline src/api/axios.ts]: Hard coding the production URL seems very nasty, can we not get something a but more dynamic in here?

File: src/api/axios.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const local = 'http://localhost:8080' // 'https://wps-api-dev-6-secure-auzhsi.pathfinder.gov.bc.ca'
+// const baseURL = isBundled ? '{{.Env.API_BASE_URL}}' : local
+const baseURL = isBundled ? prod : local
+
+const instance = axios.create({ baseURL })
+
+// instance.interceptors.response.use(
```
Comment (Sybrand) [inline src/api/axios.ts]: Do we need this commented out code?

File: src/components/PageHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export const PageHeader = () => {
+  return (
+    <nav>
+      <Container>
+        <a href="https://www2.gov.bc.ca">
+          <img
+            src={process.env.PUBLIC_URL + '/images/bcid-logo-rev-en.svg'}
+            width="181"
```
Comment (Sybrand) [inline src/components/PageHeader.tsx]: Should width and height not be coming from stylesheets?

File: src/features/fwiCalculator/components/StationsDropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  return (
+    <>
+      <Autocomplete
+        data-testid="weather-station-dropdown"
+        options={stations}
+        getOptionLabel={option => `${option.name}(${option.code})`}
+        onChange={onChange}
+        style={{ width: 300 }}
```
Comment (Sybrand) [inline src/features/fwiCalculator/components/StationsDropdown.tsx]: Shouldn't this width come from a stylesheet?

File: src/api/axios.ts (hunk 2 above)
Comment (Kyubinhan) [reply src/api/axios.ts]: Yes eventually we need to pull urls from the environment variable but it was kept like that for the time being

**Pull request merged**